src/sundialsUtils.cpp

```diff
@@ -86,9 +86,9 @@ void SundialsCvode::initialize()
             );
         }
         sundialsLinsol = SUNDenseLinearSolver(y.forSundials(), (SUNMatrix) sundialsLinsolMatrix);
-        CVodeSetLinearSolver(sundialsMem, (SUNLinearSolver) sundialsLinsol,
+        CVDlsSetLinearSolver(sundialsMem, (SUNLinearSolver) sundialsLinsol,
                                 (SUNMatrix) sundialsLinsolMatrix);
-        flag = CVodeSetJacFn(sundialsMem, denseJac);
+        flag = CVDlsSetJacFn(sundialsMem, denseJac);
```

Comment:
The names starting with `CVode` are the more modern versions -- the `CVDls...` methods are deprecated aliases.